### PR TITLE
Launch App

### PR DESCRIPTION
**Pull request by darwin808**

### CODE REVIEW

**Review by ardeay — CHANGES_REQUESTED** [review]
image
Edit single page for marketplace items to check if its a type "template", if so, have the button open up "/launch/?template=ZUID"

File: src/pages/launch.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        });
+    } catch (error) {
+      ErrorMsg({
+        title: error.message,
+        text: error?.response.data?.message,
+      });
+    }
+  };
+
+  const handleDownload = async (e) => {
```
Comment (ardeay) [inline src/pages/launch.js:96]: This functionality will belong in accounts under each instance

instances/ZUID/settings - add a card at bottom that says "export as template" which when the button is clicked, it creates a ZIP file

**Review by ardeay — CHANGES_REQUESTED** [review]

File: src/pages/launch.js (hunk 1 above)
Comment (darwin808) [reply src/pages/launch.js:96]: image

**Comment by darwin808** [comment]
@ardeay 

i perform a get request using the `meta.zuid` but i dont see the template zuid example `14-xxx` 

image

**Comment by darwin808** [comment]
please see for update https://github.com/zesty-io/website/pull/1085

**Pull request closed without merging**